packages/features/bookings/components/event-meta/Occurences.tsx

```diff
@@ -22,6 +28,32 @@ export const EventOccurences = ({ event }: { event: PublicEvent }) => {
 
   if (!event.recurringEvent) return null;
 
+  if (bookerState === "booking" && recurringEventCount && selectedTimeslot) {
```

Comment:
These changes ensure that we show the selected recurring timeslots instead of the input field when the user is in the "booking" state.